Applications/Hermes/Readme.md

```diff
@@ -0,0 +1,105 @@
+# Hermes - Message Application
+
+Hermes is a simple messaging application developed for Faraday. Hermes was the God of messengers so it seemed fitting.
+
+The messaging functionality is the result of a Flask server that allows Transmit/Receive capability for locally connected Faraday devices. When the Flask server is running, requests sent to the specific port of operation allow the transmission and reception of data messages wirelessly between two (or more) Faraday devices.
+
+The `hermesflaskexample_rx.py` and `hermesflaskexample_tx.py` scripts are examples of simple user interface programs utilizing the Flask messaging server and provide functionality to transmit text between two units.
+
+### Prerequisites
+* Properly configured and connected proxy
+  * x2 Faraday connected to local computer
+ 
+> Note: Keep the units separated a few feet apart and ensure the RF power settings are below ~20 to avoid de-sensing the CC430 front end receiver!
+
+> Note: Until proxy auto-configuration functionality is added it is possible that proxy's assigned callsign is different than the units actual configuration callsign. Please keep these matching unless you know what you're doing.
+
+An example hermes.ini with two units connected is shown below.
+
+```Python
+[FLASK]
+HOST=127.0.0.1
+PORT=8005
+
+[PROXY]
+UNITS=2
+
+[UNIT0]
+CALLSIGN=kb1lqd
```

Comment:
Actually completely removed per example further in file and to remove redundant example.